variables.tf

```
@@ -455,14 +455,14 @@ variable "metadata_http_put_response_hop_limit" {
 
 variable "metadata_http_tokens_required" {
   type        = bool
-  default     = true
+  default     = false
```

Why change this default ?

```suggestion
  default     = true
```